src/main/java/core/basesyntax/NumberBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class NumberBall {
+    public int getRandomNumber() {
+        Random randomNumber = new Random();
+        int value = randomNumber.nextInt(100);
+        return value;
```

Comment:
The temporary variable 'value' is not necessary here. You can directly return the result of 'randomNumber.nextInt(100)'.